tmol/score/disulfide/potentials/params.hh

```diff
@@ -0,0 +1,120 @@
+#pragma once
+
+#include <Eigen/Core>
+
+#include <tmol/utility/tensor/TensorAccessor.h>
+#include <tmol/utility/tensor/TensorUtil.h>
+
+namespace tmol {
+namespace score {
+namespace disulfide {
+namespace potentials {
+
+template <typename T, int N>
+using Vec = Eigen::Matrix<T, N, 1>;
+
+template <typename Real>
+struct DisulfideGlobalParams {
+  Real d_location;
+  Real d_scale;
+  Real d_shape;
+
+  Real a_logA;
+  Real a_kappa;
+  Real a_mu;
+
+  Real dss_logA1;
+  Real dss_kappa1;
+  Real dss_mu1;
+  Real dss_logA2;
+  Real dss_kappa2;
+  Real dss_mu2;
+
+  Real dcs_logA1;
+  Real dcs_mu1;
+  Real dcs_kappa1;
+  Real dcs_logA2;
+  Real dcs_mu2;
+  Real dcs_kappa2;
+  Real dcs_logA3;
+  Real dcs_mu3;
+  Real dcs_kappa3;
+
+  Real wt_dih_ss;
+  Real wt_dih_cs;
+  Real wt_ang;
+  Real wt_len;
+  Real shift;
+};
+
+template <typename Real, tmol::Device D>
+struct DisulfideGlobalParamTensors {
```

Comment:
It doesn't look like this class is necessary. I don't see it used anywhere else.